api/memory/graphiti_tool.py

```
@@ -434,11 +423,11 @@
                 return similar_queries
 
             except Exception as cypher_error:
-                print(f"Error executing Cypher query: {cypher_error}")
+                logging.error("Error executing Cypher query: %s", cypher_error)
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)